### ISSUE TITLE
Differentiate verse from quotes

### ISSUE THREAD

**Issue by contrapunctus-1** [dakrone/ox-tufte#13]
`tufte-html` exports `verse` blocks as `blockquote`. I'd prefer that it behave similar to (or just delegate to) `org-html-verse-block`, so verse blocks may be styled differently from quote blocks.

https://github.com/dakrone/ox-tufte/blob/master/ox-tufte.el#L87

What do the others say about it? If it's not desirable for some reason, it's also easy to achieve this via configuration -
```elisp
(setf (alist-get 'verse-block
                 (org-export-backend-transcoders
                  (org-export-get-backend 'tufte-html)))
      'org-html-verse-block)
```

**Comment by dakrone** [comment]
I think I prefer the current behavior, but the workaround works for those that don't. I think that is good for now?

**Comment by contrapunctus-1** [comment]
Thank you for your response.

I just discovered that this was based on a misunderstanding - I thought tufte-css specified significant line-spacing for epigraphs, but it was merely me using escaped newlines (which I now realized are unnecessary) in verse blocks.

I still don't see why ox-tufte exports verse blocks using the "epigraph" class when quote blocks do that already, but my immediate issue has been resolved.
